react-ui/src/components/Layouts/InputWorkerTimes/__snapshots__/CheckBoxApplication.test.js.snap

```diff
@@ -2,6 +2,11 @@
 
 exports[`Checkbox component snapshot renders 1`] = `
 <div>
+  <h1
```

Comment:
Please note that these changes were made to ensure passing front end unit tests due to the formatting changes on this page.